### PR TITLE
Make access to PR information easier

### PR DESCRIPTION
**Pull request by AndreasBurger**
Adds getters for special git-config entries created by the pull request resource on pull request jobs.
Also adds convenience-method for fetching all labels on pull requests via GitHubRepoHelper.

### CODE REVIEW

File: github/util.py [hunk 1]
```diff
@@ -498,6 +498,10 @@ def is_org_member(self, organization_name, user_login):
         organization = self.github.organization(organization_name)
         return organization.is_member(user_login)
 
+    def fetch_labels_on_pr(self, pull_request_number):
```
Comment (ccwienk) [inline github/util.py]: nit: type hint?

File: gitutil.py [hunk 2]
```diff
@@ -38,6 +39,12 @@ def _ssh_auth_env(github_cfg):
     return tmp_id
 
 
+class ConfigLevel(enum.Enum):
+    LOCAL = 'repository'
+    GLOBAL = 'global'
+    SYSTEM = 'system'
```
Comment (ccwienk) [inline gitutil.py]: add `NONE = None` (or maybe `DEFAULT = None`?)

File: gitutil.py [hunk 3]
```diff
@@ -151,6 +158,11 @@ def _has_stash(self):
     def _pop_stash(self):
         self.repo.git.stash('pop', '--quiet')
 
+    def _config_value(self, section, option, level:ConfigLevel=None):
```
Comment (ccwienk) [inline gitutil.py]: use `NONE / DEFAULT` to get rid of the special-case below

File: gitutil.py [hunk 4]
```diff
@@ -151,6 +158,11 @@ def _has_stash(self):
...
+        if level:
+            level = level.value
```
Comment (ccwienk) [inline gitutil.py]: not required - always use level.value

btw: why is this required? I know what the cfg levels mean on git-side. just wondering why we need to discriminate now.

Comment (AndreasBurger) [reply gitutil.py]: I dont think it is needed for the introduced changes, as the api mentions
> If None, all applicable levels will be used

I'd rather do it properly, however.

File: github/util.py (hunk 1 above)
Comment (AndreasBurger) [reply github/util.py]: On further thought, I agree. I've removed it, as the benefit is miniscule
